src/main/java/core/basesyntax/service/operations/PurchaseOperation.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class PurchaseOperation implements OperationHandler {
+    @Override
+    public void getOperation(List<FruitTransaction> transactions) {
+
+        // someone has bought some fruits

```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty line.